models/recurly.yml

```diff
@@ -320,10 +320,10 @@ models:
   - name: recurly__monthly_recurring_revenue
     description: Each record represents an account and MRR generated on a monthly basis.
     tests:
-      - dbt_utils.unique_combination_of_columns:
-          combination_of_columns:
-            - account_id
-            - account_month    
+      - dbt_utils.unique_combination_of_columns: 
+          combination_of_columns: 
+            - account_id 
+            - account_month     
```

Comment:
We use a combination of columns test here, but we use a surrogate key in the others. What are your thoughts on creating a surrogate key to be consistent with the daily model?

Reply:
Agreed! Added.